### DISCUSSION TITLE
Welcome to starknet-docs Discussions!

### DISCUSSION THREAD

**Discussion by stoobie** [starknet-io/starknet-docs#963] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NicKGNT** [comment]
hi, I use the starknet ecosystem, it has a lot of differences from other L2 solutions, it will be interesting to watch the development
